src/go/rpk/pkg/cli/transform/deploy.go

```diff
@@ -34,21 +34,17 @@ func newDeployCommand(fs afero.Fs, p *config.Params) *cobra.Command {
 		Short: "Deploy a transform",
 		Long: `Deploy a transform.
 
-When run in the same directory as a transform.yaml it will read the configuration file,
-then look for a .wasm file with the same name as your project. If the input and output
-topics are specified in the configuration file, those will be used, otherwise the topics
-can be specified on the command line using the --input-topic and --output-topic flags.
+When run in the same directory as a transform.yaml, this reads the configuration file, then looks for a .wasm file with the same name as your project.
```

Comment:
Is this text 80col wrapped? Looks a bit wide

Reply:
Nope, didn't realize that was a requirement, thanks fixed!